bundles/commerce-mock-0.1.0/chart/commerce-mock/templates/mf-commerce.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: "ui.kyma-project.io/v1alpha1"
+kind: MicroFrontend
+metadata:
+  name: commercermicrofrontend
+  namespace: <namespace>

```

Comment:
```suggestion
```

Reply:
we shouldn't remove the namespace since we want it to be deployed to a certain namespace only

Reply:
removed per our conversation